Mathlib/Topology/Algebra/Constructions.lean

```diff
@@ -180,4 +180,24 @@ theorem continuous_coe_inv : Continuous (fun u => ↑u⁻¹ : Mˣ → M) :=
 #align units.continuous_coe_inv Units.continuous_coe_inv
 #align add_units.continuous_coe_neg AddUnits.continuous_coe_neg
 
+@[to_additive]
+protected lemma continuous_inv : Continuous (fun u ↦ u⁻¹ : Mˣ → Mˣ) := by
```

Comment:
We should have a `ContinuousInv` instance instead of a type-specific lemma. Then use [Homeomorph.inv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Group/Basic.html#Homeomorph.inv) instead of the next definition.

Reply:
Done!